dense_vector/challenges/default.json

```diff
@@ -27,6 +27,51 @@
       "warmup-time-period": {{ bulk_warmup | default(40) | int }},
       "clients": {{bulk_indexing_clients | default(1)}}
     },
+    {
+      "name": "refresh-after-index",
+      "operation": "refresh"
+    },
+    {
+      "name": "wait-until-merges-finish-after-index",
```

Comment:
I wonder if we can just keep this one, and drop the next one `wait-until-merges-finish`? Since force merge is blocking, not sure we need to wait for merges to finish?

Reply:
@jtibshirani Thanks for your comment.  In the past, we had a situation when merges were still happening in the background even after force merge is done. That's why now we rely on merge stats, details are [here](https://github.com/elastic/rally-tracks/commit/e2292ec2cb723e231bd8e3f4f8840088645a823a).